src/figuresClasses.ts

```diff
@@ -1,11 +1,80 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number | string;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public sideA: number,
+    public sideB: number,
+    public sideC: number,
+    public shape: string = 'triangle',
+  ) {
+    if (sideA <= 0 || sideB <= 0 || sideC <= 0) {
+      throw new Error('Arror');
+    }
+
+    if (
+      sideA + sideB <= sideC ||
+      sideA + sideC <= sideB ||
+      sideB + sideC <= sideA
+    ) {
+      throw new Error(
```

Comment:
[CODE STYLE] - The error message contains a typo ('arror') and could be more descriptive. Consider changing it to 'The provided side lengths do not form a valid triangle.'